src/HydraAuction/Delegate.hs

```diff
@@ -40,25 +36,6 @@ data DelegateEvent
   | AuctionStageStarted AuctionStage
   | HydraEvent HydraEvent
 
-newtype DelegateRunnerT m x = MkDelegateRunner
```

Comment:
Maybe we could keep around a type synonym (like: `type DelegateRunnerT m a = StateT DelegateState m a`, just in case we decide to add other stuff in the future

Reply:
Good thought, but I oppose:

1. MLabs styleguide discourages `type`s usage
2. Change gonna be simple either way, this pattern should not go in other places